content/en/blog/_posts/2021-06-18-Announcing-Kubernetes-Upstream-Community-Group-Annual-Reports.md

```diff
@@ -0,0 +1,17 @@
+
+As of today the existing mechanisms, typically, look at (devstats)[https://k8s.devstats.cncf.io/], GitHub data, issues, to measure the healthiness of a given UG/WG/SIG/Committee. As a project spanning several diverse communities, it was essential to have something that captured the human side of things. With 50,000+ contributors, it’s easy to assume that the project has enough help and this report surfaces more information than /help-wanted and /good-first-issue for end users. This is how we sustain the project. Paraphrasing one of the Steering Committee members, [Paris Pittman](https://github.com/parispittman), “There was a requirement for tighter feedback loops - ones that involved more than just Github data and issues. Given that Kubernetes, as a project, has grown in scale & number of contributors over the years, we have outgrown the existing reporting mechanisms."
+
+The existing communication channels between the Steering committee members & the folks leading the groups & committees were also required to be made as open & as bi-directional as possible. Towards achieving this very purpose, every group & committee has been assigned a liaison from among the steering committee members for kick off, help, or guidance needed throughout the process. According to [Davanum Srinivas a.k.a. dims](https://github.com/dims), “... That was one of the main motivations behind this report. People (leading the groups/committees) know that they can reach out to us and there’s a vehicle for them to reach out to us… This is our way of setting up a two-way feedback for them." The progress on these action items would be updated and tracked on the monthly Steering Committee meetings, ensuring that this is not a one-off activity. Quoting [Nikhita Raghunath](https://github.com/nikhita), one of the Steering Committee members, “... Once we have a base, the liaisons will work with these groups to ensure that the problems are resolved. When we have a report next year, we’ll have a look at the progress made & how we could still do better. But the idea is definitely to not stop at the report.”
```

Comment:
I'd use a block quote for SC member quotes (NB small other tweaks also suggested here):
```suggestion
The existing communication channels between the Steering Committee members & the folks leading the groups & committees were also required to be made as open & as bi-directional as possible. Towards achieving this very purpose, every group & committee has been assigned a liaison from among the steering committee members for kick off, help, or guidance needed throughout the reporting process.

According to [Davanum Srinivas a.k.a. dims](https://github.com/dims):
> …That was one of the main motivations behind this report. People (leading the
> groups/committees) know that they can reach out to us and there’s a vehicle for
> them to reach out to us  … this is our way of setting up a two-way feedback for them.

The progress on these action items would be updated and tracked on the monthly
Steering Committee meetings, ensuring that this is not a one-off activity.  
Quoting [Nikhita Raghunath](https://github.com/nikhita), another of the Steering Committee
members:
> Once we have a base, the liaisons will work with these groups to ensure that
> the problems are resolved. When we have a report next year, we’ll have a look
> at the progress made & how we could still do better. But the idea is definitely to
> not stop at the report.